src/lib.rs

```diff
@@ -7,6 +7,7 @@ pub mod cursor;
 pub mod data_source;
 pub mod text;
 pub mod types;
+pub mod value;
```

Comment:
Should this be `element`?

Reply:
I considered that--though we currently have a bunch of other _value_ like things lurking such as the `SymbolToken` etc.